frontend/src/components/CodeLanguageList.tsx

```diff
@@ -0,0 +1,56 @@
+import { Badge, Flex, Icon } from '@chakra-ui/react';
+
+// const CodeLanguages = ['Python', 'Javascript', 'Ruby', 'Elixir', 'C++', 'C#'];
```

Comment:
Unnecessary comment, either delete it or use it with zip function to generate the mappings below

Reply:
Moved Literal `CodeSmell` and `Language` types to module with quizData for now.